### PR TITLE
Add basestation configuration

### PR DESCRIPTION
**Pull request by Florisheinen1**
I would like to be able to alter the frequency of basestations from within Robothub. This requires a message that can get the frequency and set the frequency.
This can be done with a general configuration message, to which other things can be added besides frequency in the future if necessary.
Because there is a practical limit of teams (2), only two frequencies will be used. To ensure the basestation can handle the new frequency, and to ensure it does not collide with others, I abstracted frequency from the messages, and want to make use of predefined channels: channel 0 and channel 1, (hence only a single bit for frequency)

### CODE REVIEW

File: generator/packets.py [hunk 1]
```diff
@@ -109,5 +107,18 @@
         ["remVersion", 4, None, "Version of roboteam_embedded_messages"],
         ["id",         4, None, "Id of the robot"],
         ["message_length", 8, None, "Length of the following message"]
+    ],
+    "BasestationGetConfiguration" : [
+        ["header",     8, None, "Header byte indicating the type of packet"]
+    ],
+    "BasestationConfiguration" : [
+        ["header",     8, None, "Header byte indicating the type of packet"],
+        ["remVersion", 4, None, "Version of roboteam_embedded_messages"],
```
Comment (skaeringur97) [inline generator/packets.py:116]: remVersion should be 5 now right?

Comment (emielsteerneman) [reply generator/packets.py:116]: 4 stands for 4 bits used, not for the version

**Comment by emielsteerneman** [comment]
Looks good to me

**Review by emielsteerneman — APPROVED** [review]
Looks good to me. You can leave the REM version at 4 if you want, since this PR doesn't introduce any breaking changes.

**Pull request merged**